cmd/config/cnv-metrics.yml

```diff
@@ -0,0 +1,119 @@
+---
+# OpenShift virtualization
+## metadata
+- query: kubevirt_hyperconverged_operator_health_status{}
+  metricName: hco-operator-health-cnv
+  instant: true
+
+- query: sum(kubevirt_number_of_vms)
+  metricName: total-vm-created-cnv
+  instant: true
+
+- query: sum(cnv:vmi_status_running:count)
+  metricName: total-vmi-running-cnv
+  instant: true
+
+- query: cluster:vmi_request_cpu_cores:sum
+  metricName: total-cpu-request-cnv
+  instant: true
+
+- query: sum(kubevirt_vm_resource_requests)
+  metricName: total-mem-request-cnv
+  instant: true
+
+- query: kubevirt_virt_api_up
+  metricName: virt-api-pod-count-cnv
```

Comment:
Is this name correct? Seems like a typo

Reply:
this query tracks the number of virt-api pod running.